### DISCUSSION TITLE
AI偶像？

### DISCUSSION THREAD

**Discussion by lyien1021** [1101-GS4719-C/Python-Community-GS4719-C#1198] [General]
之前看到自己覺得蠻神奇的影片：AI偶像對另一個AI偶像團體mv的reaction
https://youtube.com/shorts/FMgxtKewIno?feature=share
還有以下是練習室影片，我覺得蠻厲害的連影子都有做出來🤣
https://youtu.be/uIdnx4TwVgU

來簡單介紹一下AI虛擬偶像的原理—「全像術」
全像術（hologram）記錄光被物體散射所形成的干擾圖形。如此我們便可事後重製散射光，製造出該物體實際存在的錯覺。捕捉全像需要雷射光源，最常使用的是紅色的氦氖雷射。先以分光鏡將光束分開，接著每道新的光束會循不同路線照射在全像記錄盤上。第一道光束──物射束（object beam）會照射在物體本身，第二道則照射在記錄盤上，作為製作干擾圖形的參考依據。

每道光束在抵達目標以前，都會通過一個鏡片。全像記錄器的鏡片有別於照相機鏡片，其目的不是要聚光，而是散光。物射束照射在物體上時，光會反射與折射，部份光束會射往全像片的方向。物射束抵達全像片後，會與參考光束（reference beam）發生碰撞，並形成干擾圖形。光敏感化學物質所製造的微粒能忠實記錄干擾圖形，其原理就如同攝影技術。

曝光的全像片基本上比較像CD，而不像攝影負片。儲存的資訊無法光靠肉眼閱讀，它看起來更像是一連串凹凸不平的波形線條。若想觀看全像圖，我們必須將光源照射在底片上。

全像片主要分為兩種，每種觀看的方式都不一樣。觀看穿透式全像片時，我們要將單色光照透底片，藉此製造出單一顏色（通常為綠色）的浮動3D圖像。倘若是在鈔票上所看到的反射式全像片，其回彈的光線則可能是多彩的。

製造全像片的過程跟傳統攝影一樣，要慎防其他光線滲入，因此我們通常必須在暗房裡方能錄製全像片。由於大部分全像雷射為紅色，傳統暗房的紅光會破壞全像片，因此必須以綠光或藍綠光取代。

全像片對環境因子也比一般攝影敏感得多。由於要將大量資訊裝載在如此小的空間裡，雷射光只要出現細微的偏差，就可能會破壞全像。地板振動或身體動作所造成的空氣振動，甚至人的呼吸都可能干擾全像製作過程。

-----
不知道大家對AI偶像的想法是什麼，我自己是覺得應該蠻難真的喜歡上一個AI偶像，畢竟他不是真的人，可能最多覺得他們的音樂好聽而當音飯？而且我很好奇之後會不會出那種AI綜藝，感覺超有趣，看AI跟AI之間如何對話🤣
而且同時我也很好奇，AI做出的那些反應是由誰來決定的，還是那是AI本身自動產生的反應？

**Comment by pigeon9975** [comment]
我第一次接觸到AI偶像是這支影片
https://www.youtube.com/watch?v=MWA9fhDYmDI&t=16s
但這已經是一年前的東西了，感覺當時的技術還沒有很成熟，裡面的人看起來表情都超僵硬，介於人和非人類之間，有點恐怖谷的感覺。
不過老實說最近做出來的東西(就是上面分享的兩部影片)那個表情還是略為驚悚，看來在成為偶像這個部分AI還有很大的進步空間！

**Comment by zilinng** [comment]
我也覺得我很難喜歡AI偶像，雖然他們有些長得真的超級好看，表演還不會翻車，但我覺得有時候太完美反而有一點冷冰冰的感覺，其實我有時候喜歡明星其實是喜歡他們私底下的互動或綜藝節目上面比較真實的一面，表演什麼的反而是其次，不過追AI偶像也有優點就是不用擔心塌房之類的事情，一想到小時候喜歡的明星現在可能在監獄裡就覺得小時候眼光怎麼可以那麼差

**Comment by tower20021202** [comment]
AI偶像真的超級酷，不僅做任何事都不會有瑕疵，而且還可以永保青春，時時刻刻保有粉絲們喜歡的樣子，真的超酷

**Comment by harrycoconut** [comment]
AI偶像對我而言，才是偶像真正的樣子！偶像不就是為了粉絲的夢想而努力編織出來的美夢嗎，AI偶像完美的詮釋了這一點，0瑕疵的表現，相信在不遠的將來與真人偶像相比也能不分軒輊😌
